env0/resource_environment.go

```diff
@@ -209,11 +209,17 @@ func setEnvironmentSchema(d *schema.ResourceData, environment client.Environment
 }
 
 func setEnvironmentConfigurationSchema(d *schema.ResourceData, configurationVariables []client.ConfigurationVariable) {
-	for index, configurationVariable := range configurationVariables {
+	var variables []interface{}
```

Comment:
I have no idea how this worked and how it had passed the acceptance tests.
Took my a while to figure out what was wrong and how to fix it.

(With TF_ACC it panics).